src/avg.rs

```diff
@@ -7,6 +8,7 @@ pub struct AVG {
     pub severity: Severity,
     pub status: Status,
     pub required_by: Vec<String>,
+    pub avg_types: HashSet<String>,
```

Comment:
mmm, use a `Vec` please. Yes, I know that you need an `HashSet` to remove the duplicates, but that's a detail of the implementation.

Reply:
done